src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,65 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Task that has a start time and an end time.
+ */
+public class Event extends Task {
+    protected String from;
+    protected String to;
+    protected LocalDateTime fromDateTime;
+    protected LocalDateTime toDateTime;
+
+    /**
+     * Constructs Event instance.
+     * @param taskName Name of task.
+     * @param from Task start time.
+     * @param to Task end time.
+     */
+    public Event(String taskName, String from, String to) {
+        super(taskName);
+        this.from = from;
+        this.to = to;
+        try {
+            this.fromDateTime = LocalDateTime.parse(from, DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm"));
+        } catch (DateTimeParseException err) {
+            this.fromDateTime = null;
+        }
+        try {
+            this.toDateTime = LocalDateTime.parse(to, DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm"));
+        } catch (DateTimeParseException err) {
+            this.toDateTime = null;
+        }
```

Comment:
Instead of using two try catch, throw a `DukeException` to handle both cases. 
Having an event from: `LocalDateTime` to: `null`, kind of defeats the purpose of the event task?

Reply:
I feel that a DukeException for an invalid time format would be very limiting for users who might want to put an abstract representation of time (e.g. a person may want to do event X from 'after lunch' to 'before dinner')

Reply:
I've actually never thought of that, cool reasoning for it!